src/main/resources/versions.json

```diff
@@ -3,7 +3,7 @@
         "8.3.0-alpha03": [
```

Comment:
isn't this also at alpha05 now?

Reply:
you are right, alpha05 is available, pushed